web/ui/react-app/src/index.tsx

```diff
@@ -6,20 +6,10 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 import { isPresent } from './utils';
 
 // Declared/defined in public/index.html, value replaced by Prometheus when serving bundle.
-declare const GLOBAL_PATH_PREFIX: string;
```

Comment:
You'll also want to remove these then:

* https://github.com/prometheus/prometheus/blob/19c190b406c992278aaade63be92ecc7bb6a4921/web/web.go#L395
* https://github.com/prometheus/prometheus/blob/19c190b406c992278aaade63be92ecc7bb6a4921/web/ui/react-app/public/index.html#L22